CMakeLists.txt

```diff
@@ -240,7 +240,9 @@ else()
       add_definitions(-DMULTIPASS_COMPILER_GCC)
     endif()
 
-    add_compile_options(-Wextra -Wempty-body -Wformat-security -Winit-self -Warray-bounds -Wnon-virtual-dtor)
+    add_compile_options(-Wextra -Wempty-body -Wformat-security -Winit-self -Warray-bounds)
```

Comment:
Why is this needed?

Reply:
The `non-virtual-dtor` is not a valid flag when compiling the `bridge_helper.c` which is why it's explicitly added to `CMAKE_CXX_FLAGS` a few lines below.

Reply:
Hmm, couldn't we just compile the bridge helper as cpp? It was already adapted for us, right?

Reply:
Meh, nevermind. Leave it